### PR TITLE
Added a GetTabSummary endpoint

### PR DESCRIPTION
**Pull request by sultan-duisenbay**
Updated the api proto and implemented RPCs and HTTP endpoint.

### CODE REVIEW

File: pkg/api/v1/summary_test.go [hunk 1]
```diff
@@ -181,3 +181,167 @@ func TestListTabSummaries(t *testing.T) {
...
+				},
+			},
+			req: &apipb.GetTabSummaryRequest{
+				Dashboard: "acme",
+				Tab:       "me-me",
+			},
+			expectError: true,
+		},
+		{
+			name: "Returns correct tab summary for a dashboard-tab",
```
Comment (michelle192837) [inline pkg/api/v1/summary_test.go:251]: nit: I think there's one more case to add (summaries exist for tabs, but you request a nonexistent tab)

File: pkg/api/v1/summary.go [hunk 2]
```diff
@@ -131,3 +134,83 @@ func (s Server) ListTabSummariesHTTP(w http.ResponseWriter, r *http.Request) {
...
+	}
+
+	if summary == nil {
+		return nil, fmt.Errorf("summary for dashboard {%q} not found.", reqDashboardName)
+	}
+
+	// TODO(sultan-duisenbay): convert fractional part of timestamp to nanos of timestamppb
+	for _, tabSummary := range summary.GetTabSummaries() {
+		if tabSummary.DashboardTabName == tabName {
+			resp := apipb.GetTabSummaryResponse{
```
Comment (michelle192837) [inline pkg/api/v1/summary.go:178]: nit: Should this be a utility function shared between this and the other function?

**Review by michelle192837 — COMMENTED** [review]
Looks mostly good, added a couple comments!

File: pkg/api/v1/summary.go (hunk 2 above)
Comment (sultan-duisenbay) [reply pkg/api/v1/summary.go:178]: Agree, done!

**Review by michelle192837 — APPROVED** [review]
Looks good, thanks! ^^

**Pull request merged**